UPGRADE.md

```diff
@@ -1,3 +1,10 @@
+2.x to 4.0
```

Comment:
```suggestion
3.x to 4.0
```

Reply:
It's sort of "any version up to 4.x"

Reply:
Ah! Gotcha. Then we can keep it like that :)